src/scene/container/RenderGroup.ts

```diff
@@ -220,6 +220,17 @@ export class RenderGroup implements Instruction
         }
     }
 
+    public destroy()
+    {
+        this.renderGroupParent = null;
+        this.root = null;
+        (this.childrenRenderablesToUpdate as any) = null;
+        (this.childrenToUpdate as any) = null;
+        (this.renderGroupChildren as any) = null;
+        (this._onRenderContainers as any) = null;
```

Comment:
Take this or leave it, but I think this will avoid the `as any` casting. 
```suggestion
        Object.assign(this, {
            childrenRenderablesToUpdate: null,
            childrenToUpdate: null,
            renderGroupChildren: null,
            _onRenderContainers: null,
        });
```